.github/workflows/main.yml

```diff
@@ -94,7 +94,7 @@ jobs:
 
       - uses: actions/setup-go@v2
         with:
-          go-version: '1.17'
+          go-version: '1.18'
```

Comment:
I _think_ this is the go version used to build prod, so we should bump it for other CI flows too.

You don't strictly need to bump it in go.mod unless you want to use features from 1.18, because well-behaved dev tooling should use the latest minor version installed anyway, but it can't hurt. And not all dev tooling is well-behaved!